pkg/operator/controller/controller_dns_node_resolver_daemonset.go

```diff
@@ -38,6 +37,27 @@ var (
 	nodeResolverScript = manifests.NodeResolverScript()
 )
 
+// buildServicesList combines the default service with additional services from the DNS spec.
+// The default service is always included first, followed by any additional services.
+func buildServicesList(dns *operatorv1.DNS) string {
+	// Start with the default service
+	services := []string{defaultService}
+
+	// Add any additional services from the spec
+	if len(dns.Spec.NodeServices) > 0 {
+		for _, service := range dns.Spec.NodeServices {
```

Comment:
@candita Will this ensure that the list, assuming items aren't changing, are in a stable order or do we need to sort them to prevent churn?